### PR TITLE
docs: Update incorrect comment for skip-update flag

### PR DESCRIPTION
**Pull request by adamcohen2**
## Description

This PR fixes an incorrect comment in the `skip-update` flag.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [ x I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [x] I've added usage information (if the PR introduces new options)
- [x] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: pkg/flag/db_flags.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ var (
 		Aliases: []Alias{
 			{
 				Name:       "skip-update",
-				Deprecated: true, // --security-update was renamed to --skip-db-update
+				Deprecated: true, // --skip-update was renamed to --skip-db-update
```
Comment (adamcohen2) [inline pkg/flag/db_flags.go:33]: This comment is for the `--skip-update` flag, so I'm not sure why it was referencing the `--security-update` flag.

**Comment by knqyf263** [comment]
Hi, could you sign our CLA?

**Comment by adamcohen2** [comment]
> Hi, could you sign our CLA?

will do, just waiting for the legal department to give me the thumbs up on your CLA.

**Comment by adamcohen2** [comment]
> Hi, could you sign our CLA?

Done!

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
Thanks!

**Pull request merged**